server/seed/redux_seed.js

```diff
@@ -0,0 +1,15 @@
+const {db, models: { Redux } } = require('../db/db')
+
+const init = async () =>{
+    try{
+        await Redux.create({id: 'IndexR1', snippet: 'import <% importName %> from "<% importFrom %>"'})
+        await Redux.create({id: 'IndexR2', snippet: 'const initialState = { <% stateName %>: <% intialStateValue 5 %>}'})
+        await Redux.create({id: 'IndexR3', snippet: 'export const reducer = combineReducers({ <% stateName %>: <% stateReducer %>,})'})
+        await Redux.create({id: 'IndexR4', snippet: 'const store = createStore(<% storeValues %>);'})
```

Comment:
Are the "%" indicating that we'd ask the user what they want their state & reducer to be called, and then we'd adjust the name of it in their boilerplate?